.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish-dry-run:
+    name: "Perform dry run for publish"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
```

Comment:
nit: sorry, I can't resist: should we use `v2` as used in the `release` job below?